sale_order_secondary_unit_price/models/sale_order_line.py

```diff
@@ -18,7 +18,7 @@ def onchange_secondary_price(self):
         if not self.secondary_uom_id:
             self.secondary_uom_price = 0
             return
-        if self.secondary_uom_price:
+        if self.secondary_uom_price is not None:
             factor = self.secondary_uom_id.factor * self.product_uom.factor
             self.price_unit = self.secondary_uom_price / factor
```

Comment:
Can we just remove the condition?
```suggestion
        factor = self.secondary_uom_id.factor * self.product_uom.factor
        self.price_unit = self.secondary_uom_price / factor
```

Reply:
Yes, we can.